pypfopt/expected_returns.py

```diff
@@ -66,8 +66,13 @@ def mean_historical_return(prices, frequency=252):
     if not isinstance(prices, pd.DataFrame):
         warnings.warn("prices are not in a dataframe", RuntimeWarning)
         prices = pd.DataFrame(prices)
-    returns = returns_from_prices(prices)
-    return returns.mean() * frequency
+    meanReturns = returns_from_prices(prices).mean()
```

Comment:
I think we are better off keeping the already used variable name (`returns`) or, at least, comply with [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) and don't use camel case (`mean_returns` instead).